bin/brew

```diff
@@ -19,6 +19,14 @@ symlink_target_directory() {
   quiet_cd "$directory" && quiet_cd "$target_dirname" && pwd -P
 }
 
+# Enable and use default Bash builtins rather than user-defined functions
+for cmd in $(compgen -A builtin)
+do
+  unset -f $cmd
```

Comment:
Could `unset` be a user- defined function?

Reply:
```sh
$ unset () { echo 1; }
$ unset foo
1
```
But I see your point :)